drv/lpc55-spi-server/src/main.rs

```diff
@@ -160,66 +143,10 @@ fn turn_on_flexcomm(syscon: &Syscon) {
     syscon.leave_reset(Peripheral::Fc3).unwrap_lite();
 }
 
-fn muck_with_gpios(syscon: &Syscon) {
-    syscon.enable_clock(Peripheral::Iocon).unwrap_lite();
```

Comment:
These `syscon` calls don't appear to exist in the generated code – is it okay to delete them?

Reply:
Yes, this was leftover from before we had the IOCON as part of the GPIO driver.